common/config/rush/pnpm-lock.yaml

```diff
@@ -18093,10 +18109,11 @@ packages:
     dev: false
 
   file:projects/perf-ai-language-text.tgz:
-    resolution: {integrity: sha512-nlS6c6fVbERUSiWQLaFMqPGWbKdeX264SUjyMVQtb7Cs4oThACUTt6R6fszcFjZlom+djlQrLrHyeYRAqhsLYw==, tarball: file:projects/perf-ai-language-text.tgz}
+    resolution: {integrity: sha512-3ur7UK0GpLa1zQRCR6qMTd/+aHtqCTwd8D4sSsmo84Me/9EdPPJW7KhnDknfWMzw562mDAj0p2cc//GYsFyF1A==, tarball: file:projects/perf-ai-language-text.tgz}
     name: '@rush-temp/perf-ai-language-text'
     version: 0.0.0
     dependencies:
+      '@azure/ai-language-text': 1.1.0-beta.2
```

Comment:
Let's update the dependency version in the perf package to be the next beta.3 and revert the changes in the lockfile.